projects/Fitlit-part-one.md

```diff
@@ -254,25 +260,23 @@ You are *not expected* to test:
 
 ### Test-Driven Development
 * 4: Application covers all aspects of the application including various flows and covers both happy/sad paths.
-* 3: Application is well tested but fails to cover some features and only tests for happy paths. Tests use smaller, sample data files as input rather than the large, original data files.
-* 2: Application makes some use of tests, but the coverage is insufficient given project requirements.
-* 1: Application does not demonstrate strong use of TDD.
+* 3: Application is well tested but fails to cover some features and only tests for happy paths. Tests use smaller, sample data files as input rather than the large, original data files. Tests must be passing to be considered.
+* 2: Application makes some use of tests, but the coverage is insufficient given project requirements.  Some tests might be failing.
+* 1: Application does not demonstrate strong use of TDD, and multiple tests might be failing.
```

Comment:
Lines 263 - 265:  Add note about passing tests.

If a student cannot get a test passing, do we want them to remove it altogether before submitting?  
Or is a failing test acceptable if they are working on it (and maybe have an issue open on their project board addressing it)?   

I like to see them open an issue for work-in-progress and therefore would be ok with a small number of failing tests.    Expecting them to remove all failing tests also sounds reasonable - what do you two think?

@sertmer @niksseif

Reply:
@hfaerber ! Failing tests are tricky, because it either indicates that they modified something in their code without taking test consideration into account, or that something is not functioning as expected. I'd rather not find any failed tests (removing them) since we also teach about red/green strategy in M1.  It also makes it easy to properly score, depending on the number of methods and tests implemented.

Reply:
@niksseif That sounds good to me.  
I left the verbiage as 'Tests must be passing to be considered.' for scores of 2, 3 & 4 and put 'some tests might be failing' for 1.    

I think it clarifies our expectation while letting us grade based on the thoroughness of their testing.   I wouldn't want to give a group a 1 if they had generally thorough testing but forgot to delete one failing test that they were struggling with.

Does that verbiage sound good to you?

Reply:
Sounds good to me - about as clear as we can make it!